srcpkgs/rabbitmq-c/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'rabbitmq-c'
+pkgname=rabbitmq-c
+version=0.9.0
+revision=1
+build_style=cmake
+hostmakedepends="popt xmlto doxygen"
+makedepends="libressl-devel"
+depends="libressl"
+short_desc="RabbitMQ C client"
+maintainer="Hans-J. Schmid <knock@myopendoor.de>"
+license="MIT"
+homepage="https://github.com/alanxz/rabbitmq-c"
+distfiles="https://github.com/alanxz/${pkgname}/archive/v${version}.tar.gz"
+checksum=316c0d156452b488124806911a62e0c2aa8a546d38fc8324719cd29aaa493024
+
+post_install() {
+ vlicense LICENSE-MIT
+}
```

Comment:
You need to define the subpackage in this template, otherwise it won't be generated. Smth like this
```
rabbitmq-c-devel _package() {
        depends="${sourcepkg}>=${version}_${revision}"
        short_desc+=" - development files"
        pkg_install() {
                vmove usr/share/man/man3
                vmove usr/include
                vmove usr/lib/pkgconfig
                vmove "usr/lib/*.a"
                vmove "usr/lib/*.so"
        }
}
```

Reply:
Ok. Thanks a lot for the hint. Didn't know how to do it. Was just making a link in the file system.

I will do it again.